cmd/ferretdb/main.go

```diff
@@ -53,8 +53,13 @@ var (
 	testConnTimeoutF = flag.Duration("test-conn-timeout", 0, "test: set connection timeout")
 )
 
-// tigrisURL is a Tigris URL. It is set in the main_tigris.go.
-var tigrisURL string
+// Tigris parameters that are set at main_tigris.go.
```

Comment:
With more and more handler-specific params, I wonder if we need to store them somewhere else but `main.go` file. What if in the future we have SQLite, Cockroach or some other DB-specific params? Our main.go will be full of them and yet for most of the users only a tiny subset will be needed.

Reply:
Well, that's internal detail that we could refactor at any time. I just don't think that time is now.

Reply:
That sounds like something we could say about any tech debt :P
But I agree, it's not for now.